scripts/firedrake-install

```diff
@@ -1014,8 +1014,16 @@ def build_and_install_petsc():
         petsc_dir, petsc_arch = get_petsc_dir()
         check_call(["rm", "-rf", os.path.join(petsc_dir, petsc_arch)])
         petsc_options = get_petsc_options()
-        check_call([python, "./configure", "PETSC_DIR={}".format(petsc_dir), "PETSC_ARCH={}".format(petsc_arch)] + petsc_options)
-        check_call(["make", "PETSC_DIR={}".format(petsc_dir), "PETSC_ARCH={}".format(petsc_arch), "all"])
+        # Set MACOSX_DEPLOYMENT_TARGET to the current macos version
+        # to workaround issues configuring PETSc on macos.
+        try:
+            MACOSX_DEPLOYMENT_TARGET = check_output(["sw_vers", "-productVersion"])
+            log.info("MACOSX_DEPLOYMENT_TARGET set to {}".format(MACOSX_DEPLOYMENT_TARGET))
```

Comment:
I think this needs to just use `MAJOR.MINOR` not `MAJOR.MINOR.POINT` (which comes back from `sw_vers`).

Reply:
Fixed.